### ISSUE TITLE
update readme gif with new functionality

### ISSUE THREAD

**Issue by mohitkyadav** [mohitkyadav/react-pagination-nav#17]

**Comment by bhattaraib58** [comment]
Can take this ??
What is the new functionality to be included in GIF?

**Comment by mohitkyadav** [comment]
first and last page buttons

**Comment by bhattaraib58** [comment]
@mohitkyadav is this connected to this issue: https://github.com/mohitkyadav/react-pagination-nav/issues/5

Seems like that issue https://github.com/mohitkyadav/react-pagination-nav/issues/5 has been fixed by https://github.com/mohitkyadav/react-pagination-nav/pull/18.

@mohitkyadav  do you want to have the gif of feature implemented by https://github.com/mohitkyadav/react-pagination-nav/pull/18  ??

**Comment by mohitkyadav** [comment]
yes @bhattaraib58 , you can get creative as well and add multiple gifs for different features.

**Comment by bhattaraib58** [comment]
@mohitkyadav I am not able to create an example project to create a GIF, do you have an example project you can share ??
